### DISCUSSION TITLE
With App bundles do you still create armv7 and intelx86?

### DISCUSSION THREAD

**Discussion by ageekhere** [airsdk/Adobe-Runtime-Support#2085] [Q&A] [answered]
Hi all, I am updating my air app to use the new App bundles, my last release used the old method.
```
armv8.apk v1.001
armv7.apk v1.002
intelx86.apk v1.003
```
I have successfully created the armv8.aab, however do I sill need to create armv7.aab and intelx86.aab?
When I review release in google console I get the warning
```
MESSAGE FOR VERSION CODE 1000002 "None the of users of this APK will be able to upgrade to any of the new APKs added in this release." 
MESSAGE FOR VERSION CODE 1000003 "None the of users of this APK will be able to upgrade to any of the new APKs added in this release." 
```

And I also get
```
Devices no longer supported
phone 1768
tablet 678
```

If I try and do
```
armv8.aab v2.001
armv7.aab v2.002
intelx86.aab v2.003
```
I get an error for armv7.aab and intelx86.aab
This APK will not be served to any users because it is completely shadowed by one or more APKs with higher version codes. Remove this APK from your release or review the targeting and version codes of the APKs that you are including in this release.

Thanks for the help

**Comment by ajwfrost** [comment, marked as answer]
The "aab" bundle format includes all of the different CPU types in it, so you don't need to create separate ones for armv7, arvm8, x86, x64 now. Just upload one aab file that that should be sufficient. But it may be worth checking that you do have the aab file created correctly - it should be a lot larger than any of the APK files, and if you open it as a zip file then there's a `BundleConfig.pb` file at the top level?

The 'no longer supported' may depend on what is being added (if it thinks there's just armv8 capabilities?) plus we've updated the minimum SDK level to API 16 - but I wouldn't have expected that to be showing so many devices. It's still a lower value than you get if you're developing with Android Studio..!

thanks

**Reply by ageekhere** [reply]
Yes it is a lot larger then the APK and it does have the BundleConfig.pb file.
All seems to be working and the app update is now live.
